### PR TITLE
DSL<->Graph conversion endpoint takes request body instead request param

### PR DESCRIPTION
**Pull request by BoykoAlex**

### CODE REVIEW

**Comment by BoykoAlex** [comment]
Not applicable to master branch. Was supposed to be created for Andy's fork.

File: spring-xd-dirt/src/main/java/org/springframework/xd/dirt/rest/ToolsController.java [hunk 1]
```diff
@@ -83,16 +82,10 @@ public DocumentParseResultResource validate(@RequestParam("definitions") String
...
-	public Map<String, Object> convertJobGrabToText(@RequestParam("graph") String graphAsString) {
-		Map<String, Object> response = new HashMap<>();
-		if (graphAsString.trim().length() == 0) {
-			response.put("text", "");
-			return response;
-		}
-		ObjectMapper mapper = new ObjectMapper();
+	@RequestMapping(value = "/convertJobGraphToText", method = RequestMethod.POST)
+	public Map<String, Object> convertJobGrabToText(@RequestBody Graph graph) {
+		Map<String, Object> response = new HashMap<String, Object>();
```
Comment (ghillert) [inline spring-xd-dirt/src/main/java/org/springframework/xd/dirt/rest/ToolsController.java]: For consistence with the statement higher up, use:

`new HashMap<>();`

File: spring-xd-dirt/src/main/java/org/springframework/xd/dirt/rest/ToolsController.java [hunk 2]
```diff
@@ -33,8 +34,6 @@
 import org.springframework.xd.dirt.stream.XDStreamParser;
 import org.springframework.xd.rest.domain.DocumentParseResultResource;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-
 /**
  * A controller for integrating with frontend tools.
```
Comment (ghillert) [inline spring-xd-dirt/src/main/java/org/springframework/xd/dirt/rest/ToolsController.java]: Add author tag

**Comment by ghillert** [comment]
LGTM - otherwise

**Comment by BoykoAlex** [comment]
Corrected the code. Thanks for taking a look :-)

**Comment by ghillert** [comment]
Merged with 3c3fa4c6217c3a43f180ee2e5e2ae54d89861333

**Pull request closed without merging**